### PR TITLE
SAK-47288 WC Grader - PDF files are opened on the same window but not embedded in Grader

### PR DESCRIPTION
**Pull request by feralvarez95**
https://sakaiproject.atlassian.net/browse/SAK-47288

### CODE REVIEW

**Review by adrianfish — DISMISSED** [review]

File: webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js [hunk 1]
```diff
@@ -114,7 +114,7 @@ export class SakaiGrader extends gradableDataMixin(SakaiElement) {
     if (!this.submittedTextMode && this._submission.submittedAttachments
           && this._submission.submittedAttachments.length > 0) {
       this.selectedAttachment = this._submission.submittedAttachments[0];
-      const preview = this.submission.previewableAttachments[this.selectedAttachment.ref];
+      const preview = this.selectedAttachment.name ? this.selectedAttachment : '';
```
[the remark this answers is not in the document]
Comment (feralvarez95) [reply webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js:117]: This show the odt, docx and odp documents properly, also images are displayed in the same screen. Currently pdf reload the screen navigating to the pdf document. As far as I can test, this.submission.previewableAttachments is never filled. I updated this jira attaching a screen record.

[the remark this answers is not in the document]
Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js:117]: It is filled in AssignmentEntityProvider:1575 if there is a preview for that attachment. If you set up the libre converter and add an attachment as a student, it creates a PDF preview copy.

[the remark this answers is not in the document]
Comment (adrianfish) [reply webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js:117]: Maybe. But I just suggest that you set up libre office and make sure you test that side of things out.

https://github.com/sakaiproject/sakai/blob/master/kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/README.md

[the remark this answers is not in the document]
Comment (feralvarez95) [reply webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js]: Ok, so after test with libre office server conversion, pdf files, images and videos was not opened by grader, it was opened in the same screen. Now, if the source are not the converted files, we need to check if files are embedable in the grader.

**Review by jesusmmp — APPROVED** [review]

**Review by juanjmerono — APPROVED** [review]

**Pull request merged**